wps-common/src/main/java/au/org/aodn/aws/util/EmailService.java

```diff
@@ -113,4 +122,76 @@ public static String getCompletedJobEmailTemplate() {
     public static String getFailedJobEmailTemplate() {
         return String.format("%s/%s", WpsConfig.EMAIL_TEMPLATES_LOCATION, WpsConfig.FAILED_JOB_EMAIL_TEMPLATE_NAME);
     }
+
+    public static String formatRequestDetail(SubsetParameters subsetParameters, String collection) {
+
+        String details = "";
+
+        if (subsetParameters != null && collection != null) {
+
+            String spatialStr = EmailService.portalFormatSpatial(subsetParameters.getBbox());
```

Comment:
Don't really need to include EmalService. here.  Its a method on this class.

Reply:
Fix